src/css/controls/flags/audioplayer.less

```diff
@@ -22,43 +22,17 @@
     }
 
     .jw-controlbar {
-        vertical-align: middle;
-        display: table;
-        height: 100%;
-        left: 0;
-        bottom: 0;
-        margin: 0;
-        width: 100%;
-        min-width: 100%;
+        position: relative; // prevents children from exploding out of this container when set too small
 
-        .jw-icon-fullscreen,
-        .jw-icon-tooltip {
+        .jw-icon-tooltip,
+            .jw-icon-inline:not(.jw-icon-volume):not(.jw-icon-playback):not(.jw-icon-fullscreen):not(jw-icon-next) {
```

Comment:
`:not` itself doesn't add specificity, but the selectors inside of `:not()` do. Separating these would reduce the specificity from `0/0/80/0` to `0/0/50/0`.